.circleci/config.yml

```diff
@@ -27,7 +28,7 @@ references:
     run:
       name: Build and push Docker image
       command: |
-        docker build -t $DOCKERHUB_REPO:$TAG .
+        docker build -t $DOCKERHUB_REPO:$TAG $DOCKERFILE_PATH
```

Comment:
do you mind utilizing CircleCI 2.1 config parameters ? (also for the tag)

Reply:
Please point me to context and examples. It seems it cannot be found easy, searching for "CircleCI 2.1 config parameters". Doesn't that mean that the entire config have to be updated?

Reply:
Sorry, it's under 2.0 but most of the features was added in 2.1, here: https://circleci.com/docs/2.0/reusing-config/